CREDITS

```diff
@@ -1,15 +1,10 @@
 Credits
 -------
 
-Lead developer:
-
-  Jukka Lehtosalo <jukka.lehtosalo@iki.fi>
-
 Core team:
 
-  Guido <guido@dropbox.com>
-  David Fisher <ddfisher@dropbox.com>
-  Greg Price <gregprice@dropbox.com>
+  Jukka Lehtosalo <jukka.lehtosalo@iki.fi>
+  Guido van Rossum <guido@python.org>
 
 Contributors (in alphabetical order, including typeshed):
```

Comment:
At least mention that this is a partial list and suggest looking at the git history for a full set of contributors.